### PR TITLE
[ACS-6581] Wrap comments into multiple lines

### PR DESCRIPTION
**Pull request by AleksanderSklorz**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/ACS-6581

**What is the new behaviour?**
Comments for version manager are visible fully and wrapped. 

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by g-jaskowski — APPROVED** [review]

File: lib/content-services/src/lib/version-manager/version-list.component.scss [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
-    &-viewport {
+    .adf-version-list-viewport {
         height: 100%;
-    }
 
-    .adf-version-list-item {
-        border-bottom: 1px solid #d8d8d8;
+        .adf-version-list-item {
+            height: auto;
+            padding-top: 23.25px;
```
[the remark this answers is not in the document]
Comment (AleksanderSklorz) [reply lib/content-services/src/lib/version-manager/version-list.component.scss]: I've changed it to 23px now

File: lib/content-services/src/lib/version-manager/version-list.component.scss [hunk 2]
```diff
@@ -1,10 +1,19 @@
 .adf-version-list {
-    &-viewport {
+    .adf-version-list-viewport {
```
Comment (MichalKinas) [inline lib/content-services/src/lib/version-manager/version-list.component.scss:2]: Generally I wonder if this is the right approach, in every other place we rather use ellipsis and title attribute to handle such cases and I think we should go with the same approach here, I would leave the current ellipsis as reported in the ticket and added a title with tooltip if it's not there already, what do you think @AleksanderSklorz

**Review by MichalKinas — CHANGES_REQUESTED** [review]

File: lib/content-services/src/lib/version-manager/version-list.component.scss (hunk 2 above)
Comment (AleksanderSklorz) [reply lib/content-services/src/lib/version-manager/version-list.component.scss:2]: I discussed that with @akashrathod28 before work on that task and he wanted solution with wrapping text. I can go with any of these solutions, just please let me know @akashrathod28 @MichalKinas which one should I take.

Comment (MichalKinas) [reply lib/content-services/src/lib/version-manager/version-list.component.scss:2]: @akashrathod28 we can discuss this with Agnieszka but wrapping the text in this case would be inconsistent with other places that we have in application

Comment (AgnieszkaOpilska) [reply lib/content-services/src/lib/version-manager/version-list.component.scss:2]: @AleksanderSklorz @akashrathod28 I prefer consistent solutions in the app, so I give my vote to ellipsis and tooltip as @MichalKinas suggested.

Comment (AleksanderSklorz) [reply lib/content-services/src/lib/version-manager/version-list.component.scss:2]: thank you @AgnieszkaOpilska for answer, I will go with tooltip ald ellipsis approach

Comment (akashrathod28) [reply lib/content-services/src/lib/version-manager/version-list.component.scss:2]: I discussed with @AleksanderSklorz  we will go with tooltip approach.

Comment (AleksanderSklorz) [reply lib/content-services/src/lib/version-manager/version-list.component.scss:2]: done, I've pushed correction

**Review by MichalKinas — APPROVED** [review]

File: lib/content-services/src/lib/version-manager/version-list.component.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (AleksanderSklorz) [reply lib/content-services/src/lib/version-manager/version-list.component.scss]: FYI, now approach is completely changed so different styles are applied

**Review by dominikiwanekhyland — APPROVED** [review]

**Pull request merged**